reinforcement_learning/Intro-to-RL-Challenge-draft.ipynb

```diff
@@ -0,0 +1 @@
2+Yf/zHfzSvvPKKkWQ2bNgQtf2JJ54wbrfbvPrqq+a//uu/zN/+7d+a8ePHm9OnT0dqZs+ebaZOnWref/998/vf/95ceuml5o477ojzkSRWZWWleeGFF8y+fftMQ0ODueGGG0xJSYnp7OyM1Nxzzz2muLjYbN261Xz00UdmxowZ5q//+q8j2/v6+szkyZNNRUWF2b17t3njjTdMXl6eqampScQhJcR//ud/mt/+9rfmj3/8o2lsbDQ//vGPTVpamtm3b58xhjEcrA8++MBcdNFFpqyszPzgBz+IrGc8++eRRx4xl19+uWlubo4sx48fj2xnHBMr6QLK1VdfbaqrqyOvQ6GQKSoqMrW1tQnsytr+MqCEw2Hj8XjMU089FVnX3t5unE6n+eUvf2mMMebAgQNGkvnwww8jNZs2bTI2m8189tlncevdalpbW40kU1dXZ4z5YtzS0tLMyy+/HKn5+OOPjSRTX19vjPkiLNrtduPz+SI1q1evNi6XywSDwfgegIVccMEF5uc//zljOEgdHR3msssuM1u2bDHf+ta3IgGF8ey/Rx55xEydOvWs2xjHxEuqSzw9PT3atWuXKioqIuvsdrsqKipUX1+fwM6Sy5EjR+Tz+aLG0e12q7y8PDKO9fX1ysnJ0fTp0yM1FRUVstvt2rlzZ9x7tgq/3y9Jys3NlSTt2rVLvb29UWM5ceJElZSURI3llClTVFBQEKmprKxUIBDQ/v3749i9NYRCIa1fv15dXV3yer2M4SBVV1dr7ty5UeMm8Ts5UJ988omKiop08cUXa/78+WpqapLEOFpBUn1Z4IkTJxQKhaJ+GSSpoKBABw8eTFBXycfn80nSWcfxzDafz6f8/Pyo7ampqcrNzY3UnG/C4bCWLVuma665RpMnT5b0xTg5HA7l5ORE1f7lWJ5trM9sO1/s3btXXq9X3d3dys7O1oYNG1RaWqqGhgbGcIDWr1+vP/zhD/rwww+/tI3fyf4rLy/X2rVrNWHCBDU3N2vlypX65je/qX379jGOFpBUAQVIpOrqau3bt0/vvPNOoltJShMmTFBDQ4P8fr9+85vfaOHChaqrq0t0W0nn6NGj+sEPfqAtW7YoPT090e0ktTlz5kR+LisrU3l5ucaNG6df//rXysjISGBnkJLsLp68vDylpKR8aRZ1S0uLPB5PgrpKPmfG6uvG0ePxqLW1NWp7X1+f2trazsuxXrp0qTZu3Ki3335bY8eOjaz3eDzq6elRe3t7VP1fjuXZxvrMtvOFw+HQpZdeqmnTpqm2tlZTp07Vs88+yxgO0K5du9Ta2qq/+qu/UmpqqlJTU1VXV6fnnntOqampKigoYDwHKScnR9/4xjd06NAhfi8tIKkCisPh0LRp07R169bIunA4rK1bt8rr9Saws+Qyfvx4eTyeqHEMBALauXNnZBy9Xq/a29u1a9euSM22bdsUDodVXl4e954TxRijpUuXasOGDdq2bZvGjx8ftX3atGlKS0uLGsvGxkY1NTVFjeXevXujAt+WLVvkcrlUWloanwOxoHA4rGAwyBgO0MyZM7V37141NDRElunTp2v+/PmRnxnPwens7NThw4dVWFjI76UVJHqW7kCtX7/eOJ1Os3btWnPgwAGzePFik5OTEzWLGl/M8N+9e7fZvXu3kWT+9V//1ezevdv86U9/MsZ8cZtxTk6Oee2118yePXvMzTfffNbbjK+88kqzc+dO884775jLLrvsvLvNeMmSJcbtdpvt27dH3Yp46tSpSM0999xjSkpKzLZt28xHH31kvF6v8Xq9ke1nbkWcNWuWaWhoMJs3bzZjxow5r25FfPDBB01dXZ05cuSI2bNnj3nwwQeNzWYzv/vd74wxjOG5+vO7eIxhPPvr/vvvN9u3bzdHjhwx7777rqmoqDB5eXmmtbXVGMM4JlrSBRRjjPnZz35mSkpKjMPhMFdffbV5//33E92S5bz99ttG0peWhQsXGmO+uNX4Jz/5iSkoKDBOp9PMnDnTNDY2Ru3j5MmT5o477jDZ2dnG5XKZ733ve6ajoyMBR5M4ZxtDSeaFF16I1Jw+fdp8//vfNxdccIHJzMw0f/d3f2eam5uj9vM///M/Zs6cOSYjI8Pk5eWZ+++/3/T29sb5aBLnH/7hH8y4ceOMw+EwY8aMMTNnzoyEE2MYw3P1lwGF8eyf2267zRQWFhqHw2EuvPBCc9ttt5lDhw5FtjOOiWUzxpjEnLsBAAA4u6SagwIAAM4PBBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5BBQAAGA5/x/bvNC/+bHONAAAAABJRU5ErkJggg==\n"},"metadata":{}}]},{"cell_type":"markdown","source":["### Now you know how to **train an agent** and visualize its performance!\n","\n","___\n","\n","For extra practice, find a [environment](https://gymnasium.farama.org/environments/classic_control/) that interests you and train an agent that use one of these [algorithms](https://stable-baselines3.readthedocs.io/en/master/guide/algos.html). Keep in mind that not *all* of the *algorithm* work for *all* the *environment*!"],"metadata":{"id":"JN5RAzg3Dyvl"}},{"cell_type":"code","source":["### YOU TRY!!!"],"metadata":{"id":"_4RsJvqCEgaM"},"execution_count":null,"outputs":[]}]}
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    model = #TODO
</pre><p>Define the PPO model for them since it hasn't been covered in the learning resources yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>